src/test/ripple-libpp_demo.cpp

```diff
@@ -178,7 +171,189 @@ int main (int argc, char** argv)
     auto const allPass = std::all_of(passes.begin(), passes.end(),
         [](auto p) { return p; });
     assert(allPass);
-    std::cout << (allPass ? "All checks pass.\n" : "Some checks fail.\n");
+    std::cout << (allPass ?
+        "All single signing checks pass.\n" :
+        "Some single signing checks fail.\n");
+
+    return allPass;
+}
+
+//------------------------------------------------------------------------------
+
+// Demonstrate multisigning.
+
+// Helper function that throws if for some reason we can't create a seed.
+ripple::Seed getSeed (std::string const& seedText)
+{
+    // WARNING!
+    // Never use ripple::parseGenericSeed() for secure code.  Call
+    // ripple::randomSeed() instead, since it is cryptographically secure.
+    boost::optional<ripple::Seed> const possibleSeed {
+        ripple::parseGenericSeed (seedText)};
+
+    if (! possibleSeed)
+        throw std::runtime_error (
+            std::string ("Could not generate seed from: ") + seedText);
```

Comment:
Unless you're going to make a case with an invalid `seedText`, change this to an `assert`.
(And I don't just say that to remove the codecov miss... Ok, yeah, I do.)